MartinSmith-Lemures/hw2/Array_hw.rb

```diff
@@ -0,0 +1,58 @@
+    puts "Find the length of strings in an array"
+    
+# Version 1
+    def length_finder(input_array)
+      input_array.map! {|i| i.length}
+    end
+    
+#version 2
+    def length_finder(input_array)
+      destination = []
+      input_array.each do |i|
+        destination << i.length
+      end
+      p destination
+    end
```

Comment:
Very cool that you created two different versions for the same function. Try giving them a slightly different name, though, so when you run your code it knows which function to work with.